test/test_build_representation.ml

```diff
@@ -0,0 +1,62 @@
+module Build = Representation.Build
+module Step = Representation.Step
+module Client = Ocaml_ci_api.Client
+module Run_time = Ocaml_ci_client_lib.Run_time
+
+let test_to_json (jobs, build_status, build_created_at, expected) =
+  let result =
+    Build.to_json
+    @@ Build.from_jobs_status ~jobs ~build_status ~build_created_at
+  in
+  Alcotest.(check string) "to_json" expected result
```

Comment:
Going via a `string` could be problematic in the future. 
Ideally this would have `Alcotest.(check Build.t)` and be able to show a nice diff of the data structure. However that requires some changes to alcotest https://github.com/mirage/alcotest/issues/194

Reply:
C’est noté 👍🏾